<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CombineProject</class>
 <widget class="QWidget" name="CombineProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1370</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_left" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>385</width>
     <height>632</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_left {
background: #FFFFFF;
    border-top-left-radius: 10px;
    border-top-right-radius: 10px;
    border-bottom-left-radius: 0px;
    border-bottom-right-radius: 0px;
}
</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_right" native="true">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>0</y>
     <width>930</width>
     <height>632</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_right {
background: #FFFFFF;
    border-top-left-radius: 10px;
    border-top-right-radius: 10px;
    border-bottom-left-radius: 0px;
    border-bottom-right-radius: 0px;
}
</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_left_switch" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>632</y>
     <width>385</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_left_switch {
background: #FFFFFF;
    border-top-left-radius: 0px;
    border-top-right-radius: 0px;
    border-bottom-left-radius: 10px;
    border-bottom-right-radius: 10px;
}
</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_right_switch" native="true">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>632</y>
     <width>930</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_right_switch {
background: #FFFFFF;
    border-top-left-radius: 0px;
    border-top-right-radius: 0px;
    border-bottom-left-radius: 10px;
    border-bottom-right-radius: 10px;
}
</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
